app/models/comment.php

```diff
@@ -46,8 +56,90 @@ public function write($thread_id)
             'thread_id' => $thread_id,
             'user_id'   => $this->user_id,
             'body'      => $this->body,
+            'filepath'  => $this->filepath,
         );
 
         $db->insert('comment', $params);
     }
+
+    public static function sortMostComments()
+    {
+        $db = DB::conn();
+
+        $thread_id = $db->rows('SELECT COUNT(*), thread_id FROM comment GROUP BY thread_id ORDER BY COUNT(*) DESC, date DESC LIMIT 10');
```

Comment:
Will rename po to. $trending_threads since it will select the thread_id of most commented threads